packages/ubuntu_bootstrap/lib/services/installer_service.dart

```diff
@@ -16,13 +16,16 @@ class InstallerService {
     final status =
         await monitorStatus().firstWhere((s) => s?.isLoading == false);
     if (status?.interactive == true) {
-      // Use the default values for a number of endpoints
-      // for which a UI page isn't implemented yet.
+      // Use the default values for a number of endpoints that aren't used in
+      // the bootstrap stage, or for which a UI page isn't implemented yet.
       await _client.markConfigured([
+        'active_directory',
+        'identity',
         'mirror',
         'proxy',
         'ssh',
         'snaplist',
+        'timezone',
```

Comment:
Is it enough to mark those models as configured? I suspect we might need to adjust the `POSTINSTALL_MODEL_NAMES` in subiquity among other things since subiquity seems to create a default 'ubuntu' user if the identity model is left untouched.
@dbungert @ogayot

Reply:
Had a chat with @ogayot - right now there's no way for the UI to tell subiquity to not create a default user. This will need some changes in subiquity and possibly cloud-init.

Reply:
Turns out cloud-init already supports that, so we'll only need a small change in subiquity